index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Apple Watch Series 7</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+      
+      <nav id="nav-bar">
+        <div class="logo">Apple</div>
+        <ul>
+          <li class="nav-link"><a href="#About">About</a></li>
+          <li class="nav-link"><a href="#How-it-works">How it works</a></li>
+          <li class="nav-link"><a href="3contact">Contact</a></li>
+        </ul>
+      </nav>
+      <h1>The future of tech is here</h1>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil, aliquid.</p>
+      <img id="header-img" src="img/AS7.jpg" alt="Apple-Series-7">
+    </header>
+    <section id="About">
+      <p class="about">About The Products</p>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere libero cum tempora molestias recusandae at placeat accusantium, culpa quam, tempore odio enim sequi explicabo, ipsum nostrum voluptates hic dolore. Enim?</p>
```

Comment:
Please the section lacks a heading. consider using “h2”-“h6” elements to headings to all sections, or else use a “div” element instead for any cases where no heading is needed.

Reply:
- [ ] Please add WHY?